### ISSUE TITLE
Verify SPA + ESRI Layout Merge / Functionality

### ISSUE THREAD

**Issue by ferrx** [swinkler149/901K-Sandbox#25]
Ed- could you verify that the view I added is working properly. It should add the map to the #mapDiv 

https://github.com/swinkler149/901K-Sandbox/blob/master/views/index_swf.html

Estimate: 30 seconds

**Comment by eahollin** [comment]
do you have it running in your own vm, or do you mean for me to check out the project and run it on mine?

**Comment by ferrx** [comment]
yeah- run it on yours if you can since you have everything configured already (not 100% if my config is up to date with yours, can check on that soon). i just wanted to verify that the map still showed up correctly in the #myDiv element.
